homeassistant/components/nest/__init__.py

```diff
@@ -187,6 +192,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             entry, unique_id=entry.data[CONF_PROJECT_ID]
         )
 
+    async_delete_issue(hass, DOMAIN, "deprecated_app_auth")
```

Comment:
Where is this issue created?

Reply:
Should have been `removed_app_auth`, missed during rename.